### PR TITLE
Add column indexers.

### PR DESCRIPTION
**Pull request by degustaf**

### CODE REVIEW

File: packages/web/src/js/element/region.ts [hunk 1]
```diff
@@ -113,6 +112,41 @@ export const oddInfo: ElementInfo = {
...
+        const cells = idxMapToKeysArray(value || {});
+        for (const indexer of cells) {
+            const indexerValue = digits[indexer];
+            if (null == indexerValue) continue;
+            console.log(indexer, indexerValue);
+            const [ c, r ] = cellIdx2cellCoord(indexer, grid);
+            const indexee = cellCoord2CellIdx([ indexerValue - 1, r ], grid);
+            const indexeeValue = digits[indexee];
+            console.log(r, c, indexee, indexeeValue);
+            if (null == indexeeValue) continue;
```
Comment (wooferzfg) [inline packages/web/src/js/element/region.ts:136]: Can remove the console logs:
```suggestion
            if (null == indexerValue) continue;
            const [ c, r ] = cellIdx2cellCoord(indexer, grid);
            const indexee = cellCoord2CellIdx([ indexerValue - 1, r ], grid);
            const indexeeValue = digits[indexee];
            if (null == indexeeValue) continue;
```

File: packages/web/src/js/element/region.ts [hunk 2]
```diff
@@ -113,6 +112,41 @@ export const oddInfo: ElementInfo = {
...
+export const columnIndexerInfo: ElementInfo = {
+    getInputHandler(ref: StateRef, grid: Grid, svg: SVGSVGElement): InputHandler {
+        return getInputHandler(ref, grid, svg, '');
+    },
+    order: 42,
+    inGlobalMenu: false,
+    menu: {
+        type: 'select',
+        name: 'Column Indexer',
+        icon: 'odd-even',
```
Comment (wooferzfg) [inline packages/web/src/js/element/region.ts:124]: Just want to double check - you're intentionally using the odd/even icon, right? (I don't think we have an icon for 159 anyway)

**Review by wooferzfg — COMMENTED** [review]
Looks great!

File: packages/web/src/js/element/region.ts (hunk 1 above)
Comment (degustaf) [reply packages/web/src/js/element/region.ts:136]: Grumble, grumble, grumble. Can't believe I missed that.

File: packages/web/src/js/element/region.ts (hunk 2 above)
Comment (degustaf) [reply packages/web/src/js/element/region.ts:124]: Yes, because I didn't know what else to use.

Comment (wooferzfg) [reply packages/web/src/js/element/region.ts:124]: Makes sense! We can always update it later when we have an icon specifically for indexers.

**Pull request merged**